src/test/ui/process-termination/process-termination-blocking-io.rs

```diff
@@ -0,0 +1,19 @@
+// program should terminate even if a thread is blocked on I/O.
+// https://github.com/fortanix/rust-sgx/issues/109
+
+// run-pass
+// ignore-emscripten no threads support
+
+use std::{net::TcpListener, sync::mpsc, thread};
+
+fn main() {
+    let (tx, rx) = mpsc::channel();
+    thread::spawn(move || {
+        let listen = TcpListener::bind("0:0").unwrap();
```

Comment:
Should probably specify `0.0.0.0:0` here

Reply:
Thanks, fixed